payload-examples/package.json

```diff
@@ -12,7 +12,9 @@
     "index.json"
   ],
   "scripts": {},
-  "dependencies": {},
+  "dependencies": {
+    "@octokit/webhooks-types": "^3.70.0"
```

Comment:
might this dependency cause us trouble later? Say we release v4, we'll have to remember to bump the version here manually, right?

Reply:
We could have `semantic-release` update the version whenever a new release is cut, and make sure that `@octokit/webhooks-types` is published first

Reply:
yeah I think that would be good, if it's not too much trouble to set it up